<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>667</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>671</width>
      <height>561</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>GraphViewer</string>
     </attribute>
     <widget class="qview" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>531</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="rubberBandSelectionMode">
       <enum>Qt::ContainsItemBoundingRect</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Render View</string>
     </attribute>
     <widget class="opengl" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Data view</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>521</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>371</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>File nodes</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Niflib nodes</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>491</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Bsa</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>491</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>double click on item to edit name</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>583</y>
      <width>1051</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionNew_Scene"/>
   </widget>
   <widget class="QMenu" name="menuItems">
    <property name="title">
     <string>Items</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionAdd_Link"/>
    <addaction name="actionEdit_Link_properties"/>
    <addaction name="actionEdit_Node_properties"/>
    <addaction name="actionAdd_node_from_selected"/>
    <addaction name="actionDelete_selected"/>
    <addaction name="actionClear_all_selection_in_lists"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenfile"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionLoad_nif_xml"/>
   </widget>
   <addaction name="menuScene"/>
   <addaction name="menuItems"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Scene">
   <property name="text">
    <string>new Scene</string>
   </property>
  </action>
  <action name="actionAdd_Node">
   <property name="text">
    <string>add Node</string>
   </property>
  </action>
  <action name="actionAdd_Link">
   <property name="text">
    <string>add Link</string>
   </property>
  </action>
  <action name="actionEdit_Link_properties">
   <property name="text">
    <string>edit Link properties</string>
   </property>
  </action>
  <action name="actionEdit_Node_properties">
   <property name="text">
    <string>edit Node properties</string>
   </property>
  </action>
  <action name="actionOpenfile">
   <property name="text">
    <string>Openfile</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionAdd_node_from_selected">
   <property name="text">
    <string>add node from selected niflib list</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="text">
    <string>delete selected</string>
   </property>
  </action>
  <action name="actionClear_all_selection_in_lists">
   <property name="text">
    <string>clear all selection in lists</string>
   </property>
  </action>
  <action name="actionLoad_nif_xml">
   <property name="text">
    <string>load nif.xml</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>opengl</class>
   <extends>QWidget</extends>
   <header>opengl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qview</class>
   <extends>QGraphicsView</extends>
   <header>qview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
